tests/extensions/sample_plugins.py

```diff
@@ -0,0 +1,162 @@
+            "help_text": "Use sandbox",
+            "label": "Use sandbox",
+        },
+    }
+
+    @classmethod
+    def _get_default_configuration(cls):
+        return {
+            "name": "PluginSample",
+            "description": "Test plugin description",
+            "active": True,
+            "configuration": [
+                {"name": "Username", "value": "admin"},
+                {"name": "Password", "value": "123"},
+                {"name": "Use sandbox", "value": False},
+            ],
+        }
+
+    def calculate_checkout_total(self, checkout, discounts, previous_value):
+        total = Money("1.0", currency=checkout.get_total().currency)
+        return TaxedMoney(total, total)
+
+    def calculate_checkout_subtotal(self, checkout, discounts, previous_value):
+        subtotal = Money("1.0", currency=checkout.get_total().currency)
+        return TaxedMoney(subtotal, subtotal)
+
+    def calculate_checkout_shipping(self, checkout, discounts, previous_value):
+        price = Money("1.0", currency=checkout.get_total().currency)
+        return TaxedMoney(price, price)
+
+    def calculate_order_shipping(self, order, previous_value):
+        price = Money("1.0", currency=order.total.currency)
+        return TaxedMoney(price, price)
+
+    def calculate_checkout_line_total(self, checkout_line, discounts, previous_value):
+        price = Money("1.0", currency=checkout_line.get_total().currency)
+        return TaxedMoney(price, price)
+
+    def calculate_order_line_unit(self, order_line, previous_value):
+        currency = order_line.unit_price.currency
+        price = Money("1.0", currency)
+        return TaxedMoney(price, price)
+
+    def get_tax_rate_type_choices(self, previous_value):
+        return [TaxType(code="123", description="abc")]
+
+    def show_taxes_on_storefront(self, previous_value: bool) -> bool:
+        return True
+
+    def taxes_are_enabled(self, previous_value: bool) -> bool:
+        return True
+
+    def apply_taxes_to_product(self, product, price, country, previous_value, **kwargs):
+        price = Money("1.0", price.currency)
+        return TaxedMoney(price, price)
+
+    def apply_taxes_to_shipping(
+        self, price, shipping_address, previous_value
+    ) -> TaxedMoney:
+        price = Money("1.0", price.currency)
+        return TaxedMoney(price, price)
+
+    def get_tax_rate_percentage_value(
+        self, obj: Union["Product", "ProductType"], country: Country, previous_value
+    ) -> Decimal:
+        return Decimal("15.0").quantize(Decimal("1."))
+
+
+class Plugin2Inactive(BasePlugin):
+    PLUGIN_NAME = "Plugin2Inactive"
+    CONFIG_STRUCTURE = {}
+
+    @classmethod
+    def _get_default_configuration(cls):
+        return {
+            "name": "Plugin2Inactive",
+            "description": "Test plugin description_2",
+            "active": False,
+            "configuration": [],
+        }
+
+
+class Active(BasePlugin):
```

Comment:
Maybe something more realistic, like `ActiveTaxPlugin` or at least `ActivePlugin`?
Also - `defaults` in `description` has `"Not working"`. Is it expected?

Reply:
This description was copy-pasted so I didn't change it :)